src/hooks/useOnAvatarPress.ts

```diff
@@ -158,140 +168,89 @@ export default () => {
 
   const callback = useCallback(
     async (buttonIndex: number) => {
-      if (buttonIndex === 0) {
-        if (isENSProfile) {
-          if (!isReadOnly) {
-            onAvatarEditProfile();
-          } else {
-            onAvatarViewProfile();
-          }
+      if (isENSProfile) {
+        if (isReadOnly) {
+          if (buttonIndex === 0) onAvatarViewProfile();
```

Comment:
I would avoid 'if`s w/o brackets

Reply:
In principle I agree but I must admit it looks a bit more readable to me in this particular case 🤔